src/test/WorldIDIdentityManagerCalculation.t.sol

```diff
@@ -14,10 +14,13 @@ contract WorldIDIdentityManagerCalculation is WorldIDIdentityManagerTest {
     /// @notice Tests whether it is possible to correctly calculate the `inputHash` to the merkle
     ///         tree verifier.
     function testCalculateIdentityRegistrationInputHashFromParametersOnKnownInput() public {
-        // Setup
-        bytes memory callData = abi.encodeCall(
-            ManagerImpl.calculateIdentityRegistrationInputHash,
-            (startIndex, preRoot, postRoot, identityCommitments)
+        // Setup0
+        bytes memory callData = abi.encodeWithSelector(
+            ManagerImpl.calculateIdentityRegistrationInputHash.selector,
+            startIndex,
+            preRoot,
+            postRoot,
+            identityCommitments
```

Comment:
Why the change to `encodeWithSelector` everywhere? It's far more brittle and un-checked by the compiler.